model/src/main/java/com/ge/predix/acs/rest/ResourceRequest.java

```diff
@@ -0,0 +1,49 @@
+package com.ge.predix.acs.rest;
+
+import java.util.Set;
+
+import com.ge.predix.acs.model.Attribute;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+/**
+ * Represents a Requested Resource in the system identified by a subjectIdentifier.
+ */
+@ApiModel(description = "Represents a managed protected resource for V2.")
+public class ResourceRequest {
```

Comment:
Is RequestedResource a better name?  to avoid confusion that this is a HTTP request payload.